### PR TITLE
add Prometheus

### PR DESCRIPTION
**Pull request by rluria14**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by rluria14** [comment]
@conda-forge/help-go having some troubles getting windows to build. any suggestions?

**Comment by sodre** [comment]
You can skip it for now. I'm having an overall problem with the go compilers on Windows.

File: recipes/prometheus/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    - m2-coreutils  # [win]
+
+  commands:
+    - prometheus --version
+
+about:
+  home: https://prometheus.io/
+  license: Apache-2.0
+  license_file: {{ pkg_src }}/LICENSE
+  summary: Prometheus, a Cloud Native Computing Foundation project, is a systems and service monitoring system. It collects metrics from configured targets at given intervals, evaluates rule expressions, displays the results, and can trigger alerts if some condition is observed to be true.
```
Comment (sodre) [inline recipes/prometheus/meta.yaml]: The `summary` should usually be short. You can use the `description` field to add additional details.

Comment (rluria14) [reply recipes/prometheus/meta.yaml]: Thanks, I have shortened the summary.

File: recipes/prometheus/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  license_file: {{ pkg_src }}/LICENSE
+  summary: Prometheus, a Cloud Native Computing Foundation project, is a systems and service monitoring system.
+  doc_url: https://prometheus.io/docs/introduction/overview/
+  dev_url: https://github.com/prometheus/{{ name }}
+
+extra:
+  recipe-maintainers:
+    - rluria14
+    - adbeda
+    - mwilson8
```
[the remark this answers is not in the document]
Comment (mwilson8) [reply recipes/prometheus/meta.yaml]: @sodre yes, I agree

[the remark this answers is not in the document]
Comment (adbeda) [reply recipes/prometheus/meta.yaml]: yes

[the remark this answers is not in the document]
Comment (rluria14) [reply recipes/prometheus/meta.yaml]: @sodre thanks, all maintainers have agreed.

**Pull request merged**